### DISCUSSION TITLE
Copilot activation failed and saying that Please upgrade your Copilot extension to continue using this service.

### DISCUSSION THREAD

**Discussion by ChrisPorter87** [community/community#18443] [Copilot Conversations]
I am running the current version of Copilot (v1.17.5850) in VS Code 2.  For several weeks it worked completely fine, however as of late it completely stopped functioning.  I've tried to enable/disable and  install/uninstall, but nothing has enabled it to work again.  Has anyone else run into this issue ? As soon as I try to start typing within a .js file, the icon in the status bar in the button switches to having a slash through it and tells me that Copilot activation failed and Please upgrade your Copilot extension to continue using this service, even though I have the current version.  So confused

**Comment by Zerotask** [comment]
For me it's still working.

I could assume that other extensions are causing the problem. Can you try to deactivate all extensions except Copilot?

**Reply by ChrisPorter87** [reply]
It's telling me Copilot activation failed Please upgrade your Copilot extension to continue using this service, even though I have the current version, and under the extensions it says that it's enabled. Don't get it

**Comment by ChrisPorter87** [comment]
<img width="629" alt="Screen Shot 2022-06-12 at 1 19 51 PM" src="https://user-images.githubusercontent.com/96667529/173245252-3e3f6303-c7cd-432c-ab6e-81f1d03ac1ff.png">

**Comment by ChrisPorter87** [comment]
<img width="940" alt="Screen Shot 2022-06-12 at 1 19 43 PM" src="https://user-images.githubusercontent.com/96667529/173245255-49ed16e2-5f7d-47e8-be76-591f8d626394.png">

**Reply by trenutoo** [reply]
The current version is v1.28.6134
![image](https://user-images.githubusercontent.com/10975908/173247702-04206842-53bf-49f5-ae2d-fd11068d11be.png)

Click on the arrow next to uninstall button and make sure you're not locked into a specific older version. You also have to ensure vscode is not outdated.

**Comment by ChrisPorter87** [comment]
I can't figure out how to delete this discussion entirely, but I got it back in action somehow.  Thanks yall

**Reply by trenutoo** [reply]
> I can't figure out how to delete this discussion entirely, but I got it back in action somehow. Thanks yall

There's no need to delete discussions. Many other users might experience similar issues and will also show up in the Google index. Posting the solution that solved the issue will help others. You can mark the discussion as solved.

**Comment by devmustafa4** [comment]
For me updating VS Code solved the issue.

**Reply by johnnieskywalker** [reply]
I updated, but still experience that issue :/

**Comment by drok** [comment]
I was using vscode Version: 1.66.2, and got the same error as above, "copilot/1.17.5850" being out of date, even though it was just installed, and the vscode market place lists Copilot v1.65.7705 as current.

Something in vscode or in the extension, dependencies are not set right.

After updating vscode to the current (v1.74.2 at this time), the following prompt appears:

"GitHub Copilot requires VS Code 1.57+. Please update your VS Code"

I then had to **uninstall copilot** (which was marked as 'preview' in vscode 1.74.2), and re-install it from the marketplace.

Then, it worked as advertised.

## What worked for me:

 * Update your vscode, then
 * Uninstall "Github Copilot" extension,
 * Reinstall "Github Copilot" extension.
